src/Microsoft.FeatureManagement/FeatureManagementError.cs

```diff
@@ -8,6 +8,11 @@ public enum FeatureManagementError
         /// <summary>
         /// A feature filter that was listed for feature evaluation was not found.
         /// </summary>
-        MissingFeatureFilter
+        MissingFeatureFilter,
+
+        /// <summary>
+        /// Multiple feature filters match the name listed for feature evaluation.
+        /// </summary>
+        ConflictingFeatureFilter
```

Comment:
I understand where the 'Conflicting' is from. However, from the user's perspective, I specified a feature filter, which cannot be matched. It's not because the feature filter I specified conflicts with anyone, but because the feature filter I specified is ambiguous, for example, I should have namespaced it to be more specific etc. Therefore, I suggest we name this something like `AmbiguousMatchFeatureFilter`, similar to what .net has in [AmbiguousMatchException](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.ambiguousmatchexception?view=netcore-3.1). The intellisense can be updated accordingly.

Reply:
I was considering ambiguous as well since that's what .NET throws when a multiple controller actions match. Good idea. I think we can shorten it to `AmbiguousFeatureFilter`